.travis.yml

```diff
@@ -0,0 +1,69 @@
+language: cpp
+sudo: required
+dist: trusty
+
+addons:
+  apt:
+    packages:
+      - language-pack-en
+      - zip
+      - unzip
+      - curl
+      - git
+      - build-essential
+      - cmake
+      - automake
+      - autoconf
+      - libtool
+      - libboost-all-dev
+      - libasio-dev
+      - ssh
+      - pdsh
+      - openjdk-8-jdk-headless
+      - libmysqlclient-dev
+      - libmysql-java
+      - sysv-rc-conf
+      - maven
+      - libkrb5-dev
+      - libssl-dev
+      - libsasl2-dev
+      - libsasl2-modules-gssapi-mit
+      - libsqlite3-dev
+      - libtidy-0.99-0
+      - libxml2-dev
+      - libxslt-dev
+      - libldap2-dev
+      - python-setuptools
+      - libgmp3-dev
+      - libffi-dev
+      - ant
+      - libcppunit-dev
+      - libtool
+      - libc6-dbg
+
+services:
+  - mysql
```

Comment:
`mysql` is bootstrapped in the provisioning step but I'm not sure if it's a required dependency to run the unit tests. @eddiedugan?